README.md

```diff
@@ -1,2 +1,75 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+  
+
 # Leaderboard
- The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+All data is preserved thanks to the external. [Leaderboard API service.](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3)
+
+  
+
+[live version](https://elasfarc.github.io/Leaderboard/dist/)
+
+  
+
+![screen](https://raw.githubusercontent.com/elasfarc/Leaderboard/styling/ss3.png)
```

Comment:
- [ ] **OPTIONAL**
Please update the live version to the latest updates of the app design. It looks awesome great job 😉